recipe/meta.yaml

```
@@ -1,35 +1,41 @@
-{% set version = "0.7.13" %}
+{% set version = "0.11.2" %}
 
 package:
   name: opencensus
   version: {{ version }}
 
 source:
   url: https://github.com/census-instrumentation/opencensus-python/archive/v{{ version }}.zip
-  sha256: 53183a1f1fe952eaf7150c4d713bd16e580e55c021d5cca47df22ee822490c89
+  sha256: 7835c0f3b1aaddb27aaebf434e1b7866841ff1a9185c4c36200dd0631ca9d06b
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.6
-    - opencensus-context ==0.1.2
-    - google-api-core >=1.0.0, <2.0.0
+    - python
+    - opencensus-context >=0.1.3
+    - google-api-core >=1.0.0, <3.0.0
```

```suggestion
    - google-api-core >=1.0.0,<3.0.0
```
Redundant whitespace

---

Fixed.